src/trie/rune_util.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __RUNE_UTIL_H__
+#define __RUNE_UTIL_H__
+
+#include "../dep/libnu/libnu.h"
+
+/* Internally, the trie works with 16/32 bit "Runes", i.e. fixed with unicode
+ * characters. 16 bit shuold be fine for most use cases */
+#ifdef TRIE_32BIT_RUNES
+    typedef u_int32_t rune;
+    #define TRIE_RUNE_MASK 0xffffffff
+#else // default - 16 bit runes
+    typedef u_int16_t rune;
+    #define TRIE_RUNE_MASK 0x0000ffff
```

Comment:
it's unrelated to the mask. nunicode returns 1 or more 32 bit runes, and we take only the first one, as in `ß => ss`. In this case we'll take only "s" which is, well, it might work in this case? But it probably won't in others. We need to either change the logic or avoid normalizing in this case. But it won't cause crashes or anything, and let's remember 99% of the time we're dealing with `A=>a` etc.

BTW I've opened a feature request for nunicode to support folding accents and diacritics, and that might require deeper changes. but for now we are not 100% correct but reasonably good.